.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [opened, synchronize, reopened]
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1.3'
+          - '1'
+        os:
+          - ubuntu-latest
+          - macOS-latest
```

Comment:
We can build on Windows too, which would allow to get rid of appveyor.

Reply:
In trying to save Github Action minutes, I'd rather keep using Appveyor for Windows builds, even if it's quite slow.

Reply:
I thought GH actions were free for public repositories?

Reply:
Free within some limits. We get 2000 minutes / month / organization, and each macOS minute counts for 10. Each Windows minute counts for 2.

Reply:
According to [GitHub docs](https://docs.github.com/en/free-pro-team@latest/actions/reference/usage-limits-billing-and-administration#about-billing-for-github-actions):
> GitHub Actions usage is free for public repositories and self-hosted runners. For private repositories, each GitHub account receives a certain amount of free minutes and storage, depending on the product used with the account.

JuMP-dev has moved all their CI to GH actions. Their monthly use is well above 2000 minutes, and AFAIK they are not paying anything.

Reply:
Is that really the case? It would be much easier, but I find their wording confusing. What does "usage is free" mean? Unlimited build minutes?

Reply:
What I understand is:
* public repositories -> unlimited, free builds
* private repositories -> free up to 2,000min, then you have to pay

both subject to [usage limits](https://docs.github.com/en/free-pro-team@latest/actions/reference/usage-limits-billing-and-administration#usage-limits) such as max concurrent jobs, max job duration, max matrix size, etc.

Reply:
You're right. I got confirmation from Github too. I really don't know why we were under the impression that build time was limited. it's infinitely easier if we can use Actions across the board!